src/ol/render/canvas.js

```diff
@@ -8,6 +8,66 @@ import LRUCache from '../structs/LRUCache.js';
 import {create as createTransform} from '../transform.js';
 
 
+/**
+ * @typedef {{fillStyle: module:ol/colorlike~ColorLike}} FillState
+ */
+
+
+/**
+ * @typedef {Object} FillStrokeState
+ * @property {module:ol/colorlike~ColorLike} [currentFillStyle]
+ * @property {module:ol/colorlike~ColorLike} [currentStrokeStyle]
+ * @property string{} [currentLineCap]
```

Comment:
This should be `{string}` instead of `string{}`.

Reply:
Thanks